File: src/main/java/me/arasple/mc/trmenu/action/acts/ActionCommandConsole.java
package me.arasple.mc.trmenu.action.acts;

import io.izzel.taboolib.util.Commands;
import me.arasple.mc.trmenu.TrMenu;
import me.arasple.mc.trmenu.action.base.AbstractAction;
import org.bukkit.Bukkit;
import org.bukkit.entity.Player;

/**
 * @author Arasple
 * @date 2019/12/22 10:22
 */
public class ActionCommandConsole extends AbstractAction {

    @Override
    public String getName() {
        return "console";
    }

    @Override
    public void onExecute(Player player) {
        Bukkit.getScheduler().runTask(TrMenu.getPlugin(), () -> Commands.dispatchCommand(Bukkit.getConsoleSender(), getContent(player)));
    }

}

File: src/main/kotlin/me/arasple/mc/trmenu/modules/action/impl/ActionCommandConsole.kt
package me.arasple.mc.trmenu.modules.action.impl

import io.izzel.taboolib.util.Commands
import me.arasple.mc.trmenu.modules.action.base.Action
import me.arasple.mc.trmenu.utils.Tasks
import org.bukkit.Bukkit
import org.bukkit.entity.Player

/**
 * @author Arasple
 * @date 2020/3/8 21:17
 */
class ActionCommandConsole : Action("console") {

    override fun onExecute(player: Player) = getSplitedBySemicolon(player).forEach {
        Tasks.run {
            Commands.dispatchCommand(Bukkit.getConsoleSender(), it)
        }
    }

}